app/models/bet_option.rb

```diff
@@ -3,4 +3,5 @@ class BetOption < ApplicationRecord
   has_many :user_bets
 
   scope :winners, -> { where(winner: true) }
+  scope :losers, -> { where(winner: false) }
```

Comment:
Technically this commit needs to be earlier in the history for all of your commits to work. But dont worry about fixing it.

In one of your previous commits you try and use this scope before you define it